Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs

```diff
@@ -114,7 +114,7 @@ private void ExportUserFormModule(string path)
 
             var tempFile = ExportToTempFile();
             var tempFilePath = Directory.GetParent(tempFile).FullName;
-            var contents = File.ReadAllLines(tempFile);
+            var contents = File.ReadAllLines(tempFile,System.Text.Encoding.UTF7);
```

Comment:
are you sure that UTF-7 is the correct encoding? I'd have expected some system-dependent codepage... (also space after comma) :)

Reply:
I tested different encodings. With utf-8 the error documented in #3641 did not show up any more.